spec/rubocop/cop/factory_bot/create_list_spec.rb

```diff
@@ -138,17 +145,6 @@
       RUBY
     end
 
-    it 'flags usage of n.times with keyword arguments' do
-      expect_offense(<<~RUBY)
-        5.times { create :user, :trait, key: val }
-        ^^^^^^^ Prefer create_list.
-      RUBY
-
-      expect_correction(<<~RUBY)
-        create_list :user, 5, :trait, key: val
-      RUBY
-    end
-
```

Comment:
memo) Attempting to correct this false positive would make this case not an offense. However, I thought it would be more painful for the user to have a false positive in the case we are fixing this time than a false negative case. What do you think?

Reply:
I don't think there is a difference between 
```
 5.times { create :user, :key: val }
```
and 
```
 5.times { create :user, :trait, key: rand }
```

as we don't know if val is deterministic method call. So I guess this example was wrong to begin with